src/Microsoft.Win32.Registry/tests/RegistryKey/RegistryKey_OpenSubKey_str_rkpc.cs

```diff
@@ -41,6 +41,7 @@ public void NegativeTests()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46, "11332")]
```

Comment:
I think I prefer a full link to github over just the id so that it is clear what it is referring to.